Patch for libiconv-1.6.1+NetBSD 1.5
===================================

This is a patch file for libiconv-1.6.1 to make it work on NetBSD 1.5.

To apply this patch, you should go to the top directory of the libiconv
source tree (where you see `README' file), then invoke `patch' command
like this:

	% patch -p0 < this-file

Then follow the normal build instructions described in README and
INSTALL.generic.


diff -c -r ../libiconv-1.6.1.org/autoconf/ltconfig ./autoconf/ltconfig
*** ../libiconv-1.6.1.org/autoconf/ltconfig	Mon Feb 26 21:52:17 2001
--- ./autoconf/ltconfig	Fri Jun  1 09:29:30 2001
***************
*** 1188,1198 ****
  
    netbsd*)
      if echo __ELF__ | $CC -E - | grep __ELF__ >/dev/null; then
-       archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname -o $lib'
-       archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
-     else
        archive_cmds='$LD -Bshareable $libobjs $deplibs $linkopts -o $lib'
        # can we support soname and/or expsyms with a.out? -oliva
      fi
      ;;
  
--- 1188,1198 ----
  
    netbsd*)
      if echo __ELF__ | $CC -E - | grep __ELF__ >/dev/null; then
        archive_cmds='$LD -Bshareable $libobjs $deplibs $linkopts -o $lib'
        # can we support soname and/or expsyms with a.out? -oliva
+     else
+       archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname -o $lib'
+       archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
      fi
      ;;
  
diff -c -r ../libiconv-1.6.1.org/libcharset/autoconf/ltconfig ./libcharset/autoconf/ltconfig
*** ../libiconv-1.6.1.org/libcharset/autoconf/ltconfig	Mon Feb 26 21:49:31 2001
--- ./libcharset/autoconf/ltconfig	Fri Jun  1 09:29:31 2001
***************
*** 1194,1204 ****
  
    netbsd*)
      if echo __ELF__ | $CC -E - | grep __ELF__ >/dev/null; then
-       archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname -o $lib'
-       archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
-     else
        archive_cmds='$LD -Bshareable $libobjs $deplibs $linkopts -o $lib'
        # can we support soname and/or expsyms with a.out? -oliva
      fi
      ;;
  
--- 1194,1204 ----
  
    netbsd*)
      if echo __ELF__ | $CC -E - | grep __ELF__ >/dev/null; then
        archive_cmds='$LD -Bshareable $libobjs $deplibs $linkopts -o $lib'
        # can we support soname and/or expsyms with a.out? -oliva
+     else
+       archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname -o $lib'
+       archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
      fi
      ;;
  
